apps/website/components/vertical-nav/README.md

```diff
@@ -48,23 +48,26 @@ Vertical navigation has a few layout options including dividers and section head
 ::: inset Basic
 ![Basic Vertical Navigation](/images/components/vertical-nav/basic.png)
 :::
-**Basic**
+**Basic**<br/>
+No dividers or section headers.
 
 </div>
 <div class="clr-col-12 clr-col-md-4">
 
 ::: inset Dividers
 ![Dividers](/images/components/vertical-nav/divider.png)
 :::
-**Dividers**
+**Dividers**<br/>
```

Comment:
Does it need `<br/>` tag?

Placing empty space above and below `**Dividers**` does the trick usually.